webrender/src/device/gfx/device.rs

```diff
@@ -1390,15 +1394,14 @@ impl<B: hal::Backend> Device<B> {
                     std::mem::transmute::<_, &[u32; 17]>(&self.bound_locals)
                 }
             );
-            return;
         }
 
-        self.locals_descriptors.as_mut().unwrap().bind_locals(
+        self.locals_descriptors.bind_locals(
             self.bound_locals,
```

Comment:
Since we wan't to use 1 dummy descriptor set for Locals (which we have to bind only once) if we support push constants, this should be:
`if self.use_push_constants { Locals::default() } else { self.bound_locals } `